mlir/test/Dialect/OpenMP/invalid.mlir

```diff
@@ -874,7 +874,7 @@ func.func @omp_atomic_update(%x: memref<i32>, %expr: i32) {
 // -----
 
 func.func @omp_atomic_capture(%x: memref<i32>, %v: memref<i32>, %expr: i32) {
-  // expected-error @below {{expected three operations in omp.atomic.capture region}}
+  // expected-error @below {{expected three operations in atomic.capture region}}
```

Comment:
nit: please add testcases in `OpenACC/invalid.mlir`.

Reply:
Done :)